tools/cli/commands/changelog.js

```diff
@@ -55,3 +57,43 @@ export async function changeloggerCli( argv ) {
 		shell: true,
 	} );
 }
+
+/** Validate arguments
+ *
+ * @param {object} argv - arguments passed to changelogger.
+ */
+function validateArgs ( argv ) {
```

Comment:
Unsure if switches are best practice for verification here, but laying foundation in case we need to do anything with the arguments.